README.md

```diff
@@ -2,30 +2,50 @@
 
 # Stormpath Shiro Web Sample
 
-Fork of the Apache Shiro Web sample application that uses Stormpath for Identity Management.
+Fork of the Apache Shiro Web sample application that uses Stormpath for User Management. This sample application uses the [Apache Shiro plugin for Stormpath](https://github.com/stormpath/stormpath-shiro) to demonstrate how to integrate Apache Shiro and Stormpath.
 
-This sample application uses the [Stormpath Shiro](https://github.com/stormpath/stormpath-shiro) plugin to demonstrate how to achieve Stormpath and Shiro integration.
+Stormpath is a User Management API that reduces development time with instant-on, scalable user infrastructure. Stormpath's intuitive API and expert support make it easy for developers to authenticate, manage, and secure users and roles in any application. The `stormpath-shiro` plugin allows a [Shiro](http://shiro.apache.org/)-enabled application to use [Stormpath](http://www.stormpath.com) for all authentication and access control needs.
 
-The `stormpath-shiro` plugin allows a [Shiro](http://shiro.apache.org/)-enabled application to use the [Stormpath](http://www.stormpath.com) User Management & Authentication service for all authentication and access control needs.
+## Documentation
 
-Usage documentation [is in the plugin's wiki](https://github.com/stormpath/stormpath-shiro/wiki).
+Stormpath offers usage documentation and support for the Apache Shiro Plugin for Stormpath [in the wiki](https://github.com/stormpath/stormpath-shiro/wiki). Please email support@stormpath.com with any errors or issues with the documentation.
 
-### Build Instructions ###
+## Links
+
+Below are some resources you might find useful!
+- [The Apache Shiro Plugin for Stormpath](https://github.com/stormpath/stormpath-shiro)
+- [Stormpath-Shiro Sample Webapp](https://github.com/stormpath/stormpath-shiro-web-sample)
```

Comment:
This is the `stormpath-shiro-web-sample` and you are auto-referencing it in this line. Is this intended?